vector/src/main/java/im/vector/app/features/roomdirectory/roompreview/RoomPreviewViewModel.kt

```diff
@@ -1,263 +1,258 @@
-/*
```

Comment:
is the diff in this file correct?

Reply:
That's another part of CLRF problem, missed that, thanks

Reply:
check now, I've updated in the latest commit